modules/apps/site/site-cms-site-initializer-test/src/testIntegration/java/com/liferay/site/cms/site/initializer/internal/display/context/test/FilesSectionDisplayContextTest.java

```diff
@@ -117,6 +140,77 @@ public void testGetCreationMenu() throws Exception {
 			expectedResultMap);
 	}
 
+	@Test
```

Comment:
OPTIONAL: Couln't we add this assertions in testGetCreationMenu in oder to avoid having 2 methods (since it is faster)?